plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/DockerInstanceProvider.java

```diff
@@ -115,6 +117,41 @@ public DockerInstanceProvider(DockerConnector docker,
         }
     }
 
+
+    /**
+     * Escape paths for Windows system with boot@docker according to rules given here :
+     * https://github.com/boot2docker/boot2docker/blob/master/README.md#virtualbox-guest-additions
+     * @param paths
+     * @return escaped set of path
+     */
+    private Set<String> escapePaths(Set<String> paths) {
+        return paths.stream().map(s -> escapePath(s)).collect(Collectors.toSet());
+    }
+
+    /**
+     * Escape path for Windows system with boot@docker according to rules given here :
+     * https://github.com/boot2docker/boot2docker/blob/master/README.md#virtualbox-guest-additions
+     * @param path
+     * @return escaped set of path
+     */
+    private String escapePath(String path) {
```

Comment:
Looks like an util method which we may need in the future.
Maybe we should move it to _Util_ class and also test.